### PR TITLE
Drafting 20.04 Release Notes

### PR DESCRIPTION
**Pull request by Socialhappiness**
working off Beta page, article is hidden for now

### CODE REVIEW

File: _articles/Pop!_OS-20.04-LTS-Release-Notes [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Customers with integrated and discrete graphics can now choose Hybrid Graphics from the battery section of the user menu. While in Hybrid Graphics mode, right clicking an application in the Activities overview offers a "Launch Using Dedicated Graphics Card" option. This enables customers to use battery-saving integrated graphics while launching specific apps on the dedicated GPU.
+
+#### Offline Upgrades
+Version upgrades allow you to download the upgrade in the background and then apply it offline when you’re ready to perform the upgrade.
+
+#### Application Settings in Top Bar
+Application indicators are now present at the top right of the desktop by default.
+
+#### Faster Pop!\_Picks
+Pop!\_Shop now loads Pop!\_Picks much faster. These are curated and preferred applications tested to provide the best experience.
```
Comment (brs17) [inline _articles/Pop!_OS-20.04-LTS-Release-Notes:54]: In pop-upgrade, we're changing this bullet point to:

* More Accessible Pop!_Picks: Pop!_Shop now loads up to 20 Pop!_Picks making highlighted applications more accessible. These are curated and preferred applications tested to provide the best experience.

I think that is more accurate, so may be a good change to make here.

Comment (Socialhappiness) [reply _articles/Pop!_OS-20.04-LTS-Release-Notes]: I updated that section, thanks!

File: _articles/Pop!_OS-20.04-LTS-Release-Notes [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#### Flatpak support with the Flathub
+Pop!\_Shop now includes Flatpak application support and the Flathub application repository by default. The source of applications is visible when viewing applications details. System76 continues to curate and package key applications such as Steam, Atom, and VS Code. Pop!\_OS repositories are prioritized to provide the best user experience for these applications.
+
+#### Hybrid Graphics
+Customers with integrated and discrete graphics can now choose Hybrid Graphics from the battery section of the user menu. While in Hybrid Graphics mode, right clicking an application in the Activities overview offers a "Launch Using Dedicated Graphics Card" option. This enables customers to use battery-saving integrated graphics while launching specific apps on the dedicated GPU.
+
+#### Offline Upgrades
+Version upgrades allow you to download the upgrade in the background and then apply it offline when you’re ready to perform the upgrade.
+
+#### Application Settings in Top Bar
```
Comment (WatchMkr) [inline _articles/Pop!_OS-20.04-LTS-Release-Notes:50]: Le's go with "Application Indicators in Top Bar".

File: _articles/Pop!_OS-20.04-LTS-Release-Notes [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+#### More Accessible Pop!\_Picks
+Pop!\_Shop now loads up to 20 Pop!\_Picks making highlighted applications more accessible. These are curated and preferred applications tested to provide the best experience.
+
+#### Theme Updates
+Dark mode is enabled by default. Default desktop wallpaper has a new design. Both defaults can be changed within the Appearance settings.
+
+### GNOME 3.36
+** New lockscreen
+** Additional performance improvements
+** New Extensions application to GNOME Shell Extensions and their settings
```
Comment (WatchMkr) [inline _articles/Pop!_OS-20.04-LTS-Release-Notes:62]: ** New Extensions application to manage GNOME Shell extensions and their settings

File: _articles/Pop!_OS-20.04-LTS-Release-Notes [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+** Better organized power off / logout section in the user menu
+** Most password dialogs now have the option to reveal the password by clicking the "eye" icon
+** App folders in the application overview can now be renamed
+** GNOME Settings sextions have been rearranved for easier navigation
```
Comment (WatchMkr) [inline _articles/Pop!_OS-20.04-LTS-Release-Notes:66]: ** GNOME Settings sections have been rearranged for easier navigation

File: _articles/Pop!_OS-20.04-LTS-Release-Notes (hunk 2 above)
Comment (Socialhappiness) [reply _articles/Pop!_OS-20.04-LTS-Release-Notes:50]: ok!

**Pull request merged**